### ISSUE TITLE
Hyperparameters to replicate the conceptual caption RN50x4 experiments.

### ISSUE THREAD

**Issue by kyleliang919** [mlfoundations/open_clip#109]
My run on 4GPUs with batch size 256 using default parameters can not replicate the 22.2% top-1 zero-shot Val accuracy on imageNet1k (claimed in readme) with the code out of the box.

![W B Chart 6_15_2022, 8_55_35 AM](https://user-images.githubusercontent.com/21994498/173872010-38ff1694-7d46-4651-a691-200a969eb18c.png)

**Comment by kyleliang919** [comment]
Thanks, let me try 1e-3, will report back later.

**Comment by zerovl** [comment]
@kyleliang919 It seems that you did not enable '--gather-with-grad', which will provide higher accuracies.

**Comment by kyleliang919** [comment]
I was able to get 21.94 with 1e-3 learning rate.
